free-programming-interactive-tutorials-en.md

```diff
@@ -252,6 +252,7 @@
 * [Python for Everybody - Interactive](https://runestone.academy/runestone/books/published/py4e-int/index.html) - Barbara Ericson
 * [Python Koans](https://github.com/gregmalcolm/python_koans)
 * [Python Tutorial](https://www.w3schools.com/python) - W3Schools
+* [Python Tutorial](https://www.tutorialspoint.com/python) - Tutorialspoint
```

Comment:
Please write nouns with the correct capitalization, "Tutorials Point" is two words in title case.

```suggestion
* [Python Tutorial](https://www.tutorialspoint.com/python) - Tutorials Point
```

Reply:
Okay sure, I will do that.